src/objective-c/GRPCClient/GRPCCall+Cronet.h

```diff
@@ -39,8 +39,7 @@
  */
 @interface GRPCCall (Cronet)
 
-+(void) setUseCronet:(BOOL) useCronet
-                    :(cronet_engine *)cronet_engine;
++(void)useCronet:(cronet_engine *)cronet_engine;
```

Comment:
Document that the method isn't thread-safe, and should be called before starting RPCs. Add a sample invocation with starting Cronet and getting the singleton, and the doc will be perfect.